atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CqlCluster.java

```diff
@@ -61,4 +61,10 @@ public void close() throws IOException {
                     .getTransactionTableRangesForRepair(transactionsTableInteractions);
         }
     }
+
+    public void abortTransactions(long timestamp, List<TransactionsTableInteraction> transactionsTableInteractions) {
+        try (CqlSession session = new CqlSession(cluster.connect())) {
+            new TransactionAborter(session, config).abortTransactions(timestamp, transactionsTableInteractions);
```

Comment:
nit: I would make this a static method, so we have something like `TransactionAborter.abort(session, config, timestamp, transactionsTableInteractions`

Reply:
not convinced by that as we'd then have too many parameters in the methods of TransactionAborter.

Reply:
I meant still create the object internally, just not expose that. It's a bit of a strange API to create an object just so we can immediately call its only public method exactly once, but this comment was made before I read the tests which this enables nicely so it's ok